README.md

```diff
@@ -32,7 +32,7 @@ We must setup SQS queue and S3 event notification before use this plugin.
 Simply use RubyGems:
 
     $ gem install fluent-plugin-s3 -v "~> 0.8" --no-document # for fluentd v0.12 or later
-    $ gem install fluent-plugin-s3 -v 1.0.0 --no-document # for fluentd v1.0 or later
+    $ gem install fluent-plugin-s3 -v 1.3.0 --no-document # for fluentd v1.0 or later
```

Comment:
Hmm... removing `-v 1.3.0` is more better. It installs latest version.

Reply:
Agreed. One reason I've chosen 1.3.0 since it seems to be super stable.
If we can guarantee that latest version is the best choice, let's remove version completely.